src/main/java/com/team4/testingsystem/controllers/QuestionController.java

```diff
@@ -59,4 +80,10 @@ public QuestionDTO updateQuestion(@RequestBody QuestionDTO questionDTO, @PathVar
                 .updateQuestion(questionConverter.convertToEntity(questionDTO, id), id);
         return questionConverter.convertToDTO(resultQuestion);
     }
+
+    private List<Question> convertToEntity(List<QuestionDTO> questionsDTO) {
+        List<Question> questions = new ArrayList<>();
+        questionsDTO.forEach(questionDTO -> questions.add(questionConverter.convertToEntity(questionDTO)));
```

Comment:
Why do you like so much the decision of adding entities from the stream to create an empty list?